ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -316,10 +320,11 @@ export default defineComponent({
       headerSlotLabel: computed((): string => {
         return localState.searchType === UIMHRSearchTypes.MHRMHR_NUMBER ? 'Registration Number' : localState.searchType
       }),
-      personOrOrgLabel: computed((): string => {
-        return getManufacturedHomeSearchResults.value?.searchQuery.type === APIMHRMapSearchTypes.MHRORGANIZATION_NAME
-          ? 'Organization'
-          : 'Owner'
+      ownerOrOrgHeader: computed((): string => {
+        const found = getManufacturedHomeSearchResults.value.results
+        if (found) {
+          return found[0]?.organizationName ? 'Organization' : 'Owner'
+        } else return ''
```

Comment:
So if no results are found the header will be just 'Name' of 'Status'?

Reply:
@dimak1 If there are no results the table will not be displayed, therefore no headers to display... would it be better to return undefined instead of an empty string in that case?

Reply:
In this case, maybe we can simplify to
`return getManufacturedHomeSearchResults.value?.results[0]?.organizationName ? 'Organization' : 'Owner'` since it won't matter what is showing.

Reply:
I tried to write it in shorter hand but I can't seem to get it to work when there are no results with shorthand...